Package@swift-5.5.swift

```diff
@@ -0,0 +1,77 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "swift-doc",
+    platforms: [
+        .macOS(.v10_15)
+    ],
+    products: [
+        .executable(name: "swift-doc", targets: ["swift-doc"]),
+        .library(name: "SwiftDoc", targets: ["SwiftDoc"])
+    ],
+    dependencies: [
+        .package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", .revision("release/5.5)),
```

Comment:
A typo I guess?

```suggestion
        .package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", .revision("release/5.5")),
```

Reply:
Ugh, that didn't show up in time.  Yes, that is a typo.  Thanks.